src/touchstone/databases/elasticsearch.py

```diff
@@ -163,3 +168,32 @@ def access_dotted_field(self, input_dict, fields):
             else:
                 return None
         return tmp_dict
+
+    # def gen_result_list(self, response):
+    #    output_list = []
+    #    for hit in response:
+    #        output_list.append(hit.__dict__["_d_"])
+    #    return output_list
+
+    def get_timeseries_results(self, uuid, compute_map, index, identifier):
+
+        # not aggreated data
```

Comment:
Random comment or actual comment?

Reply:
actual comment... the start of code for non-aggregated data (will change the spelling tho)